docs/usermanual/index.md

```diff
@@ -17,16 +17,159 @@ TODO
 
 ## AWS
 
-TODO
-
 ### Disk snapshot
 
-TODO
+Use case: you have a volume `vol1` in an AWS account (Account A, in zone `us-east-2b`) which you would like to make a copy 
+of in a different account (Account B).
+
+Your AWS credentials configuration (`~/.aws/credentials`) should look like this:
+
+```text
+# Access credentials to Account A
+[src_profile]
+aws_access_key_id = xxx
+aws_secret_access_key = xxx
+
+# Access credentials to Account B
+[dst_profile]
+aws_access_key_id = yyy
+aws_secret_access_key = yyy
+
+# Access credentials to the default account to use with AWS operations. Note that this can be the same account as one 
+# of the accounts above. This is the account that will be used by libcloudforensics for AWSAccount objects that are not 
+# instantiated with a particular profile.
+[default]
+aws_access_key_id = zzz
+aws_secret_access_key = zzz
+```
```

Comment:
Maybe move this to a `prerequisites` or `setup` section or something? Also fine to leave for a future PR

Reply:
Sounds good, added section